custom_components/omnik_inverter/manifest.json

```
@@ -9,6 +9,6 @@
     "@robbinjanssen",
     "@klaasnicolaas"
   ],
-  "requirements": ["omnikinverter==0.6.1"],
+  "requirements": ["git+https://github.com/mvdwetering/python-omnikinverter.git@solis_wifistick#omnikinverter==0.0.0"],
```

```suggestion
  "requirements": ["omnikinverter==0.7.0"],
```
You can change it to this version 😉